hokusai/staging.yml

```diff
@@ -48,12 +91,23 @@ metadata:
     component: web
   name: aprd-web
   namespace: default
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:iam::585031190124:server-certificate/2018-01-17_artsy-net-wildcard"
+    service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
+    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
 spec:
   ports:
   - port: 80
     protocol: TCP
     name: http
-    targetPort: 5000
+    targetPort: 80
+  - port: 443
+    protocol: TCP
+    name: https
+    targetPort: 80
```

Comment:
@izakp should this be 443 since we have https://github.com/artsy/apr-dashboard/pull/4/files#diff-9af8dd47d98e3e7912dadcf50c599d06R35 ?